dev-packages/application-package/src/extension-package.ts

```diff
@@ -25,6 +25,10 @@ export interface Extension {
     backend?: string;
     backendElectron?: string;
     electronMain?: string;
+    /**
+     * Map of script name to script relative path
+     */
+    entryPoints?: Record<string, string>;
```

Comment:
As mentioned elsewhere, I don't think we need an id for the entry points: if the _values_ are not unique, we should coalesce them anyway. (no need to load /foo/bar/zoz twice when it is mentioned twice.).

Reply:
So you suggest logic to de-duplicate entry points based on their value(/path)?

Reply:
If we're just using them as roots in the reachability graph, yes. If this was Java, I'd use a Set<String>.

Reply:
I did not bother trying to de-duplicate yet. My reasoning is that there's a low chance of duplication here.

Reply:
The point is not the deduplication, the main point is that we're requiring a unique id when we don't need one.